src/frontend/src/common/components/library/data_subview/components/EditSamplesConfirmationModal/components/ImportFile/parseFile.ts

```diff
@@ -83,7 +84,7 @@ function warnBadFormatMetadata(
 function getMissingHeaderFields(uploadedHeaders: string[]): Set<string> | null {
   const missingFields = new Set<string>();
   if (!uploadedHeaders.includes("currentPrivateID")) {
-    missingFields.add("currentPrivateID");
+    missingFields.add("Current Private ID");
```

Comment:
I think this should refer to a look-up instead -- `SAMPLE_EDIT_METADATA_KEYS_TO_HEADERS` has this info -- because if the originating key of `currentPrivateID` ever changes the human-friendly name of "Current Private ID", we can just change it in the look up instead.

Reply:
changed